lib/main.dart

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Can you delete this empty line at the top? I know it's kinda nitpicky, but I like having the imports at the very top of the file

Reply:
Just fixed